client/src/components/questionsAnswers/M-SubmissionModals.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+const { useState } = React;
+
+function SubModals(props) {
```

Comment:
More of a suggestion rather than a critical/immediate request, I recommend when naming components to choose a word with meaning that provides some context

Reply:
Submodals is an abbreviated reference to the file name "submission modals." Its coded to allow me to capture both user questions and answers, so i used that name rather than something more specific.